torch/xpu/__init__.py

```diff
@@ -395,6 +395,24 @@ def synchronize(device: _device_t = None) -> None:
     return torch._C._xpu_synchronize(device)
 
 
+def get_arch_list() -> List[str]:
+    r"""Return list XPU architectures this library was compiled for."""
+    if not is_available():
+        return []
+    arch_flags = torch._C._xpu_getArchFlags()
+    if arch_flags is None:
+        return []
+    return arch_flags.split()
```

Comment:
Thanks for your approval. We need to set env variable `TORCH_XPU_ARCH_LIST` via splitting arch name with commas [here](https://github.com/pytorch/pytorch/pull/137773/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R124) . And we have to replace commas with whitespace in `TORCH_XPU_ARCH_LIST` [here](https://github.com/pytorch/pytorch/pull/137773/files#diff-be66263b345a09046360dd9b72163a4674b436f135608f0b32b716c60d6bafc2R420). The reason is that `C10_STRINGIZE`, used in this [line](https://github.com/pytorch/pytorch/pull/137773/files#diff-3ab42dad1ad751fd118dab7aaad0c667a9631ab91b11fb68d9384b7fd4b437aeR45), is a macro which could process only an argument into a string (it will consider input separated by commas as multi-arguments, resulting a compilation error). So, `_xpu_getArchFlags` will return an arch list separated by whitespace instead of commas.